modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-values-factory/src/main/java/com/liferay/dynamic/data/mapping/form/values/factory/internal/DDMFormValuesFactoryImpl.java

```diff
@@ -148,7 +157,11 @@ private void _checkDDMFormFieldParameterNames(
 				_checkDDMFormFieldParameterNames(
 					ddmFormField.getNestedDDMFormFields(),
 					filteredDDMFormFieldParameterName,
-					ddmFormFieldParameterNames);
+					ddmFormFieldParameterNames, fullDDMFormFields);
+			}
```

Comment:
Hi @linolaoj !
I noticed this code might not run if the condition on line 139 is true.
I would like to know if it is not a problem to not run the _checkDDMFormFieldParameterNames() inside the FOR in some cases.
My comment is similar to Carol's here:  https://github.com/liferay-forms/liferay-portal/pull/2150#discussion_r1117580672